portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,39 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+body {
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.topnav {
+  overflow: hidden;
+  background-color: black;
+}
+
+.topnav a {
+  float: left;
+  color: #eeeeee;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 17px;
+}
+
+.topnav a:hover {
+  background-color: white;
+  color: black;
+}
+
+.topnav a.active {
+  background-color: black;
+  color: white;
+}
```

Comment:
You can remove `background-color: black;` since you set the background to be black for `.topnav` above.

Also, did you want the color to be different when active? I think setting it to be white vs. #eeeeee is quite similar.

Reply:
The intention was to change it to a different color when active, updated the css so it makes more sense.